src/components/nav.tsx

```diff
@@ -42,14 +42,14 @@ export function Nav({ isCollapsed, links, supportLinks }: NavProps) {
           {isCollapsed ? (
             <Tooltip delayDuration={0}>
               <TooltipTrigger asChild>
-                <h1 className="ml-[3px] px-3 text-3xl font-bold hover:cursor-pointer">s.</h1>
+                <h1 className="ml-[3px] px-3 text-3xl font-bold hover:cursor-pointer"><a href="/">s.</a></h1>
```

Comment:
Lets use the nextjs `Link` component here. Feel free to merge this work and fix this in a different place. but something we should change.

Reply:
Replaced with Next Link.